portals/publisher/source/src/app/components/Apis/Details/Configuration/DesignConfigurations.jsx

```diff
@@ -148,7 +154,24 @@ function copyAPIConfig(api) {
             accessControlAllowMethods: [...api.corsConfiguration.accessControlAllowMethods],
         },
         additionalProperties: [...api.additionalProperties],
+        type: api.type,
+        policies: [...api.policies],
+        endpointConfig: api.endpointConfig,
     };
+    if (api.advertiseInfo) {
+        copiedConfig = {
+            ...copiedConfig,
```

Comment:
check whether the copying is necessary

Reply:
removed with https://github.com/wso2/apim-apps/pull/130/commits/d8e97d81f0afdc38eeb2c8b3fade3c2c2393ef08